configure.ac

```diff
@@ -1170,17 +1170,16 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <ctime>]],
 )
 
 dnl Check for different ways of gathering OS randomness
-AC_MSG_CHECKING([for Linux getrandom syscall])
+AC_MSG_CHECKING([for Linux getrandom function])
 AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <unistd.h>
-  #include <sys/syscall.h>
-  #include <linux/random.h>]],
- [[ syscall(SYS_getrandom, nullptr, 32, 0); ]])],
- [ AC_MSG_RESULT([yes]); AC_DEFINE([HAVE_SYS_GETRANDOM], [1], [Define this symbol if the Linux getrandom system call is available]) ],
+  #include <sys/random.h>]],
+ [[ getrandom(nullptr, 32, 0); ]])],
```

Comment:
What Linux systems do require `#include <unistd.h>` for the `getrandom` call?

Reply:
None?

Reply:
Then may be drop it? (as it done c13c97dbf846cf0e6a5581ac414ef96a215b0dc6)

Reply:
Dropped.